RocketChat/Rocket.Chat.ReactNative
# fix: enables navigation for deleted threads

## Proposed changes

Enables navigation to threads where the initial message was deleted.

## Issue(s)	


## How to test or reproduce

In any channel

- Open a new thread
- Send a couple of messages using the “send to channel” feature.
- Go back to the main channel and verify the redirection of those messages.
- Delete the message that starts the thread.
- Repeat step 3.
- Inside the thread, send another message with the “send to channel” feature.
- Repeat step 3.

## Screenshots

## Types of changes




## Checklist



## Further comments

Jira: [TC-908]


[TC-908]: https://rocketchat.atlassian.net/browse/TC-908?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
